micrometer-meter-provider/src/test/java/io/opentelemetry/contrib/metrics/micrometer/internal/instruments/MicrometerDoubleCounterTest.java

```diff
@@ -324,7 +324,7 @@ void observableWithAttributes() {
     callbackRegistrar.run();
     assertThat(counter.count()).isEqualTo(20.0);
 
-    double expectedCount = 10.0;
+    double expectedCount = 20.0;
```

Comment:
value needs to be at least 20 (as above) since the monotonic value can't decrease

other tests in this file were correctly using `20`